rtl/eclrtl/rtlkey.cpp

```diff
@@ -1860,7 +1860,7 @@ void CStringSet::addRange(const void *loval, const void *hival)
         int rc;
         while ((int)a<b)
         {
-            int i = (a+b+1)/2;
+            int i = a+(b+1-a)/2;
```

Comment:
Should these technically be b-a+1 rather than b+1-a ? i.e. can the b+1 overflow and would we care if it did?